pkg/operator/config.go

```diff
@@ -58,6 +59,55 @@ type ContainerConfig struct {
 	EnableProbes  bool
 }
 
+type Annotations struct {
+	AnnotationsString string
+	AnnotationsMap    map[string]string
+}
+
+// Implement the flag.Value interface
+func (annotations *Annotations) String() string {
+	return annotations.AnnotationsString
+}
+
+// Merge labels create a new map with labels merged.
+func (annotations *Annotations) Merge(otherAnnotations map[string]string) map[string]string {
+	mergedAnnotations := map[string]string{}
+
+	for key, value := range otherAnnotations {
+		mergedAnnotations[key] = value
+	}
+
+	for key, value := range annotations.AnnotationsMap {
+		mergedAnnotations[key] = value
+	}
+	return mergedAnnotations
+}
+
+// Set implements the flag.Set interface.
+func (annotations *Annotations) Set(value string) error {
+	m := map[string]string{}
+	if value != "" {
+		splited := strings.Split(value, ",")
+		for _, pair := range splited {
+			sp := strings.Split(pair, "=")
+			m[sp[0]] = sp[1]
+		}
+	}
+	(*annotations).AnnotationsMap = m
+	(*annotations).AnnotationsString = value
+	return nil
+}
+
+// Returns an arrary with the keys of the label map sorted
+func (annotations *Annotations) SortedKeys() []string {
```

Comment:
Looks like SortedKeys isn't used, do we need to keep it?

Reply:
So true, removing it. Blind copy-paste.